src/graph/parser.ts

```diff
@@ -278,13 +301,15 @@ export class Graph {
                         break;
                     }
 
+                    case "skipCache": {
+                        break;
+                    }
+
```

Comment:
This should be removed so the user sees the invalid key error if they try to use it.

Reply:
ah okay